sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/ContainerAPITest.groovy

```diff
@@ -799,45 +810,97 @@ class ContainerAPITest extends APISpec {
         thrown(IllegalArgumentException)
     }
 
-    // TODO (alzimmer): Turn this on once paged responses are available
-    /*def "List blobs flat marker"() {
+    def "List blobs flat marker"() {
         setup:
-        for (int i = 0; i < 10; i++) {
-            PageBlobClient bu = cu.getPageBlobClient(generateBlobName())
-            bu.create(512)
+        def NUM_BLOBS = 10
+        def PAGE_SIZE = 6
+        for (int i = 0; i < NUM_BLOBS ; i++) {
+            PageBlobClient bc = cc.getPageBlobClient(generateBlobName())
+            bc.create(512)
         }
 
-        Iterator<BlobItem> response = cu.listBlobsFlat(new ListBlobsOptions().maxResults(6), null)
+        // TODO Turn this on once paged responses are available in sync client
+        /*when: "list blobs with sync client"
+        Iterator<BlobItem> response = cc.listBlobsFlat(new ListBlobsOptions().maxResults(6), null)
 
         String marker = response.body().nextMarker()
         int firstSegmentSize = response.iterator().size()
-        response = cu.listBlobsFlat(marker, null, null)
+        response = cc.listBlobsFlat(marker, null, null)
 
-        expect:
+        then:
         firstSegmentSize == 6
         response.body().nextMarker() == null
-        responseiterator()().size() == 4
-    }*/
+        responseiterator()().size() == 4*/
```

Comment:
Is there a reason this is still commented out?

Reply:
Missed reactivating it. Will be in next commit.